### ISSUE TITLE
[Idea] Whitelist channels

### ISSUE THREAD

**Issue by Eggroley** [ajayyy/SponsorBlock#547]
A feature that'll whitelist channels so that they aren't affected by sponsorblock. Mainly for channels like InternetHistorian or brutalmoose where sponsors are just extra content.

I know this idea will just seem useless or a pain to implement to some as you can just turn skip automatically off, but I find enjoyment in the automatic skipping feature and that feature tends to become annoying when it's a sponsor segments I want to watch.

Ok bye.

**Comment by ajayyy** [comment]
This already is implemented. Open up the extension popup and click the green button.

**Comment by florentdk59** [comment]
Hello! I know this is an old ticket, and i can see that the feature seems to exist today : but i'm not sure if it has ever worked properly for me. 

- I am on Firefox 135.0.1 on Windows 11
- I see a video with a skipped segment (right at the beginning of the video) from a channel that i would like to whitelist
- I open the extension options
- i click the "whitelist channel" button
- i reload the page : the sponsored block is still skipped immediately

**Comment by ajayyy** [comment]
@florentdk59 Enable " Force channel check before skipping "

**Comment by florentdk59** [comment]
> [@florentdk59](https://github.com/florentdk59) Enable " Force channel check before skipping "

Hello friend! Thank you, i didn't know about this setting. This solves my issue! Now it works perfectly.

**Comment by farzadmf** [comment]
Question: is there a way to see/modify the whitelisted channel list? I wanted to whitelist a few channels and wanted to see if the action is reversible

**Comment by Maniakov132414** [comment]
Hello there, I know this is an old post, but I have some questions about the whitelist.

1. I couldn’t find anything in the default configuration that mentions the whitelist.
2. There is no button called “whitelist” on the YouTube website.

- I am currently using Microsoft Edge Version 147.0.3912.98 (Official build) (64-bit) on Windows 11.
- I can see the SponsorBlock popup.
- I have checked options.
- I have reloaded(F5) the video on the channel I want to add to the whitelist.
- I have tried reinstalling the extension.
